src/Microsoft.Framework.Runtime/DependencyManagement/NuGetDependencyResolver.cs

```diff
@@ -222,11 +222,8 @@ public void Initialize(IEnumerable<LibraryDescription> packages, FrameworkName t
         {
             var defaultHashPath = defaultResolver.GetHashPath(package.Id, package.Version);
```

Comment:
what's the strategy for not reading sha512? are we putting the base64 hash bytes in the lock.json written by restore?